scripts-available/CDB_Helper.sql

```diff
@@ -0,0 +1,163 @@
+-- UTF8 safe and lenght aware. Find a unique identifier with a given prefix
+-- and/or suffix and withing a schema.
+CREATE OR REPLACE FUNCTION cartodb._CDB_Unique_Identifier(prefix TEXT, relname TEXT, suffix TEXT, schema TEXT DEFAULT NULL)
+RETURNS TEXT
+AS $$
+DECLARE
+  rec RECORD;
+  usedspace INTEGER;
+  ident TEXT;
+  i INTEGER;
+  origident TEXT;
+  maxlen INTEGER;
```

Comment:
Better defining a constant to avoid programming mistakes: http://www.postgresql.org/docs/9.4/static/plpgsql-declarations.html